lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
```

Comment:
Man where was this question two weeks ago when it was fresh in my mind :( 

I *think* it was because `$this->_count` is initialized to `null`, that way we could distinguish between that and 0 records... but I don't think it really matters at this point. It could probably just become `empty($this->count)`. I'll give that a try and see how the tests do.